runtime/vendor.js

```diff
@@ -34,9 +34,8 @@ async function vendor(url, outfile) {
 async function patchAssertsBundle(assertsPath) {
   return patchFile(assertsPath, (content) =>
     content
-      .replace("const { Deno  } = globalThis;\n", "")
```

Comment:
There is more than one place using the global Deno object. It's simpler to keep it.